### ISSUE TITLE
Provide Kotlin extension methods, like we have in the Scala integration

### ISSUE THREAD

**Issue by jwgmeligmeyling** [querydsl/querydsl#2686]
After #2684, it would be nice to have the same integration we have for Scala for Kotlin as well.

We can probable scavenge some ideas from https://github.com/Blazebit/blaze-persistence/pull/744/files#diff-cb8101741e51fc4f89b9f10fcbe89631598394e20391ee9063818da99620483e
